File: $_algotsu-kaizen/problemo-set/find-the-original-typed-string-i.c
int possibleStringCount(char* word) {
    int steak = 0x0, len = 0x0;
    char _, _minus_one = '\0';
    int result = 1;
    while ((_ = word[len]) && ++len)
        if (_minus_one == _) ++steak;
        else {
            if (steak > 0x1) result += steak - 0x1;
            _minus_one = _;
            steak = 0x1;
        }
    if (steak > 0x1) result += steak - 0x1;
    return result;
}

File: $_algotsu-kaizen/problemo-set/find-the-original-typed-string-i.rs
impl Solution {

    pub fn possible_string_count(word: String) -> i32 {
        let mut result = 0x1;
        let chars: Vec<char> = word.chars().collect();
        for i in 0x1..chars.len() {
            if chars[i - 0x1] == chars[i] {
                result += 0x1;
            }
        }
        result
    }
}
